client/src/components/GameArea.js

```diff
@@ -158,6 +156,7 @@ class GameArea extends Component {
             <Battlefield
               useStubs={false}
               cards={cards}
+              player={players[0]}
```

Comment:
Is the current user always the first player?

Reply:
Probably not as-is, we'll need to handle that.